garden_ai/local_data.py

```diff
@@ -1,23 +1,27 @@
 import json
 import logging
+from enum import Enum
 from typing import Dict, Union, Optional
 from pathlib import Path
 from uuid import UUID
 
 from garden_ai.gardens import Garden
 from garden_ai.pipelines import Pipeline
-from garden_ai.utils.misc import JSON
 
 LOCAL_STORAGE = Path("~/.garden").expanduser()
 LOCAL_STORAGE.mkdir(parents=True, exist_ok=True)
 
 logger = logging.getLogger()
 
 
+class ResourceType(Enum):
+    GARDEN = "gardens"
+    PIPELINE = "pipelines"
+
+
+# This guy needs a try/except
```

Comment:
🎉🚀 Woah there, partner! 🤠 Do we _actually_ need a 🛡 try/except here? 🤔 I reckon we might've 🤠 wrangled 🤠 those pesky exceptions by checking if the file 📁 exists and making sure it ain't empty 🚫🏜 before we parse it as JSON 🤓. Not quite sure what else we need to handle 🤷‍♂️, but hey, maybe using a try/except 🎯 is the better practice over them careful if statements 🧐🚦. 🎊🎈